templates/scenarios/js/m365-tab/teamsapp.yml.tpl

```diff
@@ -84,7 +84,7 @@ deploy:
 
 # Triggered when 'teamsfx publish' is executed
 publish:
-  - uses: teamsApp/validate # This action is currently skipped, will be updated in the future version.
+  - uses: teamsApp/validateManifest # Validate using manifest schema
```

Comment:
@kuojianlu - it seems no `teamsApp/validate` in m365tab's local yml? As designed or just missing one...

Reply:
Good catch. Seems missing. I will add it.